src/js/components/ServiceDetailDebugTab.js

```diff
@@ -27,17 +27,15 @@ class ServiceDetailDebugTab extends React.Component {
     }
 
     const {version, timestamp, taskId, state, message, host} = lastTaskFailure;
-    let timeStampText = 'Just now';
-    if (new Date(timestamp) > Date.now()) {
-      timeStampText = DateUtil.msToRelativeTime(timestamp);
-    }
     let taskFailureValueMapping = {
       'Task ID': this.getValueText(taskId),
       'State': this.getValueText(state),
       'Message': this.getValueText(message),
       'Host': this.getValueText(host),
-      'Timestamp': <span>{timestamp} ({timeStampText})</span>,
-      'Version': <span>{version} ({DateUtil.msToRelativeTime(version)})</span>
+      'Timestamp': <span>
+        {timestamp} (<TimeAgo time={new Date(timestamp)} />)
+      </span>,
```

Comment:
Can we add parentheses around this to be consistent with other areas of the app? So like this:
```javascript
'Timestamp': (
  ...jsx...
),
```